templates/vpc-mini.yaml

```diff
@@ -0,0 +1,450 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >-
+  Creates a VPC with public and private subnets and a managed NAT gateway.
+Parameters:
+  VpcName:
+    Description: The VPC name (i.e. Synapse-Prod)
+    Type: String
+  VpcSubnetPrefix:
+    Description: The VPC subnet prefix (i.e. 10.255.40)
+    Type: String
+    AllowedPattern: ^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){2}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$
+  PublicSubnetZones:
+    Description: Availability zones for public subnets
+    Type: List<AWS::EC2::AvailabilityZone::Name>
+    ConstraintDescription: List of Availability Zones in a region, such as us-east-1a, us-east-1b, us-east-1c
+    Default: "us-east-1a, us-east-1b, us-east-1c"
+  PrivateSubnetZones:
+    Description: Availability zones for private subnets
+    Type: List<AWS::EC2::AvailabilityZone::Name>
+    ConstraintDescription: List of Availability Zones in a region, such as us-east-1a, us-east-1b, us-east-1c
+    Default: "us-east-1a, us-east-1b, us-east-1c"
+  VpnCidr:
+    Description: CIDR of the (sophos-utm) VPN
+    Type: String
+    Default: "10.1.0.0/16"
+    AllowedPattern: ^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/(1[6-9]|2[0-8]))$
+    ConstraintDescription: CIDR block parameter must be in the form x.x.x.x/16-28
+Mappings:
+  SubnetConfig:
+    VPC:
+      CIDR: "0/24" #10.255.X.0/24 = 10.255.X.0-10.255.X.255
+    Public:
+      CIDR: "0/28" #10.255.X.0-15
+    Private:
+      CIDR: "16/26" #10.255.X.16-79
+    Public1:
+      CIDR: "80/28" #10.255.X.80-95
+    Private1:
+      CIDR: "96/26" #10.255.X.96-159
+    Public2:
+      CIDR: "160/28" #10.255.X.160-175
+    Private2:
+      CIDR: "176/26" #10.255.X.176-240
```

Comment:
shouldn't the range be `10.255.X.176-239`?

Reply:
yes, you're right... will fix